bin/kmd

```diff
@@ -27,12 +27,12 @@ def main(args):
     tmp_prefix = os.path.join(os.path.join(args.prefix), args.release, 'root')
     fakeroot = os.path.abspath(args.release)
     if args.build or not args.install:
-        komodo.make(args.pkgs, args.repo, prefix   = tmp_prefix,
-                                         dlprefix = args.cache,
-                                         builddir = args.tmp,
-                                         jobs     = args.jobs,
-                                         cmk      = args.cmake,
-                                         fakeroot = fakeroot)
+        komodo.make(args.pkgs, args.repo,   prefix   = tmp_prefix,
```

Comment:
Aligning things at non-standard levels of indenting always break down over time. How about:

```python
komodo.make(
    args.pkgs,
    args.repo,
    prefix=tmp_prefix,
    dlprefix=args.cache,
    builddir=args.tmp,
    jobs=args.jobs,
    cmk=args.cmake,
    fakeroot = fakeroot,
    )
```
```

Reply:
I see that my comments regarding white spacing and named arguments are not in line with the rest of the code and that the rest of the code is not in line with PEP-8. Do as you like ;)